tests/integration/ambient/istioctl_test.go

```diff
@@ -0,0 +1,92 @@
+//go:build integ
+// +build integ
+
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ambient
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strings"
+	"testing"
+
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/components/istioctl"
+	kubetest "istio.io/istio/pkg/test/kube"
+	"istio.io/istio/pkg/test/util/assert"
+)
+
+func TestZtunnelConfig(t *testing.T) {
+	framework.NewTest(t).
+		Run(func(t framework.TestContext) {
+			istioCtl := istioctl.NewOrFail(t, istioctl.Config{})
+			istioCfg := istio.DefaultConfigOrFail(t, t)
+			// we do not know which ztunnel instance is located on the node as the workload, so we need to check all of them initially
```

Comment:
Every ztunnel should have identical config?

Reply:
Yeah, looks like I copied this comment from another test. Will remove it